airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/strategies/cursor_pagination_strategy.py

```diff
@@ -30,6 +31,7 @@ class CursorPaginationStrategy(PaginationStrategy, JsonSchemaMixin):
     cursor_value: Union[InterpolatedString, str]
     config: Config
     options: InitVar[Mapping[str, Any]]
+    page_size: Optional[int] = None
```

Comment:
What's the use case for `page_size` in a cursor paginator? We need the method to be defined to satisfy the interface, but maybe we should just return `None`? Also useful for making it easy to throw an error in the DefaultPaginator if the developer sets a `page_size_option` for a cursor paginator.

Reply:
some APIs use a page size with a cursor (request 5 records per page and get the next page from the cursor)

Reply:
ah okay, makes sense!